### ISSUE TITLE
Missing type definitions

### ISSUE THREAD

**Issue by drdavella** [spacetelescope/asdf#300]
PR #295 introduced more stringent warnings for missing tag types. As a result, I noticed that there are schemas for `label_mapper` and `region_selector` but no associated tag implementations. It looks like `region_selector` has a tag in the JWST pipeline, but I can't find `label_mapper`.

@nden, are both of these schemas necessary? Can either of them be moved to the JWST pipeline repo? If the answer to either is "no" or "not right now", I'll implement a workaround to suppress the warnings for now.

**Comment by nden** [comment]
The tags for these are in gwcs. It was decided at the time to split the tags and schemas as the schemas represent a "standard" of which gwcs is one implementation.

**Comment by nden** [comment]
@drdavella Can you post an example warning that you see related to `label_mapper` and what triggers it.

**Comment by drdavella** [comment]
The warnings only occur on PR #295 which hasn't been merged yet. Here's an example:

```lib.linux-x86_64-3.6/asdf/tests/test_schema.py::test_schema_example[asdf/transform/regions_selector-1.1.0.yaml-example]
  /tmp/asdf-test-qci5jgit/lib.linux-x86_64-3.6/asdf/yamlutil.py:278: UserWarning: tag:stsci.edu:asdf/transform/label_mapper-1.1.0 is not recognized, converting to raw Python data structure
    "structure".format(tag_name))
  /tmp/asdf-test-qci5jgit/lib.linux-x86_64-3.6/asdf/yamlutil.py:278: UserWarning: tag:stsci.edu:asdf/transform/regions_selector-1.1.0 is not recognized, converting to raw Python data structure
    "structure".format(tag_name))
```

The PR is intended to address issue #13. The warnings only occur during the schema validation tests which touch every installed schema. The intent of the warning is to indicate that there is no tag implementation for the given schema, therefore there is no way to convert the associated data to anything other than a raw Python data structure.

I can suppress these warnings for now but it brings us back to needing to have a discussion about where tags and schemas really belong (for astropy, gwcs, and others).

**Comment by nden** [comment]
Yes , we need to discuss this.

**Comment by drdavella** [comment]
If we want to keep these schemas in asdf itself for the long run, we'll need to figure out some way to handle these warnings. I don't want to do a blanket suppression of these warnings for this test because they actually exposed some real problems that needed to be fixed.

**Comment by nden** [comment]
I guess it all depends on what is considered a "standard". Is it a set of schemas which can have different implementations? For example the `polynomial` schema can be in `asdf-standard` but there can be a tag implementation in astropy and a different one in e.g. numpy.

**Comment by drdavella** [comment]
To clarify, by "real problems" I don't necessarily mean the warnings that I included above in this thread. There were some other real bugs that that had already been fixed in this PR.

My main point is that we don't want a blanket suppression of these warnings because they have already shown value in exposing real bugs. But it seems like we do need a way to suppress warnings in the cases where there are good reasons for not having tag implementations in asdf proper.
